### ISSUE TITLE
Datasets 

### ISSUE THREAD

**Issue by zhenye234** [sajaddarabi/ContrastiveMixup#1]
Will you provide code for other datasets in your paper( Adult Blog Covertype ) in the future?

**Comment by sajaddarabi** [comment]
Hi Zhenye, 

The tabular data pipeline is very similar to MNIST, but extracts both categorical & continuous, and implements preprocessing steps for those as detailed in the paper. 

If there's an urge for me to push it, I could do some once I have some bandwidth (probably in the coming weeks)

**Comment by mortezamg63** [comment]
I am trying to reproduce the accuracy for Adult dataset. I have challenges. I am getting accuracy 79.64%. 
For this experiment, I changed these:
1- contrastivemixup.json: change dimensions for input, projection and hidden layers to 14. change number of classes to 2 because in nn_utils we use one-hot encoding.
2-keras-mnist.py: using pandas to read data from test and train files. Next, convert to numpy for using in the program.

I am not sure for this dataset the updates are  correct. I do not see another part of code to change. 

Can you tell me what i need to update in code to get the reported accuracy in the paper? What did I do wrong?
